### DISCUSSION TITLE
More GUI control for /warps and /kits

### DISCUSSION THREAD

**Discussion by mrfloris** [Zrips/CMI#7069] [Enhancement]
It's great that we have the new GUI update to Kits and Warps, a lot has improved, and it would be even nicer if we could get additional support for a few more things, just like how /homes has that.

For example: Complexity for the GUi, so we want a GUI yes no (possible, text vs GUI), but can we get one for border yes no (no meaning only next pagination, legacy style, and yes, the current setup.

And additionally, would it be possible to have special warp buttons that we can use on page 1 as index? 

in locale, i'd like to set warps: index, warps: server, warps: events .. etc.

And then add a new warp the regular way but use a param in the command line like `/cmi setwarp name -index 2`
which wont be tied to a location, but loads the 2nd or 3rd or 4th page of the warps menu (or even a group thing).

That way we can have an index page with 
[GUI] ...
```
      [server-icon (loads page 2)
      [event-icon (loads page 4)
      [donator-icon (loads group donators warps)
```
Or maybe if we have groups, that it can auto generate some groups overview page so we can show that as an index?

Anyway, it would be nice if we can kinda control a bit more ala homes.yml settings for kits and warps.

**Comment by Momshroom** [comment]
I would love the ability to have users be able to display groups of warps from a menu as floris suggested above, and the ability to remove the border with a server setting.
